scripts/linters/pylint_extensions.py

```diff
@@ -1403,6 +1440,121 @@ def process_module(self, node):
                     self.add_message(
                         'no-newline-used-at-end', line=line_num + 1)
                 is_docstring = False
+                is_docstring_arg = False
+                is_docstring_return = False
+                is_docstring_raise = False
+                is_docstring_yield = False
+            elif is_docstring:
+                current_indentation = (len(line_with_spaces) -
+                                       len(line_with_spaces.lstrip()))
+                if line.startswith('Args:'):
+                    is_docstring_arg = True
+                    is_docstring_return = False
+                    is_docstring_raise = False
+                    is_docstring_yield = False
+                    free_form_return = False
+                    description = False
+                    outer_indentation_in_spaces = current_indentation
+                elif line.startswith('Returns:'):
+                    is_docstring_arg = False
+                    is_docstring_return = True
+                    is_docstring_raise = False
+                    is_docstring_yield = False
+                    free_form_return = False
+                    description = False
+                    outer_indentation_in_spaces = current_indentation
+                elif line.startswith('Raises:'):
+                    is_docstring_arg = False
+                    is_docstring_return = False
+                    is_docstring_raise = True
+                    is_docstring_yield = False
+                    free_form_return = False
+                    description = False
+                    outer_indentation_in_spaces = current_indentation
+                elif line.startswith('Yields:'):
+                    is_docstring_arg = False
+                    is_docstring_return = False
+                    is_docstring_raise = False
+                    is_docstring_yield = True
+                    free_form_return = False
+                    description = False
+                    outer_indentation_in_spaces = current_indentation
+                elif len(line) == 0:
+                    continue
+                elif is_docstring_arg:
+                    # Parameter
```

Comment:
Add a little more detail to these comments.

Reply:
Done